backend/src/services/user/user.service.ts

```diff
@@ -62,16 +71,20 @@ class User {
 
     const user = await this.#userRepository.create({
       email,
-      fullName,
       passwordSalt,
       passwordHash,
     });
 
+    await this.#userDetailsService.upsertUserDetails(user.id, {
```

Comment:
```suggestion
    await this.#userDetailsService.create(user.id, {
```
should be a create for now

Reply:
@what1s1ove I did as you said, but can you explain to me why it was necessary to replace upsert by create and update? the variant with upsert seems to me better